docs/deploying.md

```diff
@@ -42,7 +42,12 @@ for experimenting with infrastructure changes without going through the full CI/
 Rules for use:
 
 1. Ensure that no other developer is using the environment, as there is nothing stopping changes from overwriting each other.
-1. Clean up services you create when you are done. `terraform destroy` from within the `terraform/sandbox` directory should do it.
+1. Clean up when you are done: 
+    - `terraform destroy` from within the `terraform/sandbox` directory will take care of the provisioned services
+    - Delete the routes shown in `cf routes`
+    - Delete the apps shown in `cf apps`
+    - Delete the service keys for any remaining space deployers, likely `cf dsk <space-deployer> service-account-key`
+    - Delete the space deployers still shown in `cf services`
```

Comment:
[suggestion] I think this can be simplified by pointing people to `destroy_service_account.sh`

```suggestion
    - Delete the space deployers still shown in `cf services` by running `terraform/destroy_service_account.sh -s notify-sandbox -u <space-deployer>`
```